src/player.py

```diff
@@ -1,2 +1,7 @@
 # Write a class to hold player information, e.g. what room they are in
 # currently.
```

Comment:
Both of your Room and Player classes are looking good , I look forward to seeing the items added. Remember you can also get creative!